tests/test_orientation.py

```diff
@@ -110,14 +169,15 @@
 
 
 class TestOrientationCase(unittest.TestCase):
-    @parameterized.expand(TEST_CASES)
-    def test_ornt(self, init_param, img, data_param, expected_data, expected_code):
+    @parameterized.expand(TESTS)
+    def test_ornt(self, in_type, init_param, img, data_param, expected_data, expected_code):
+        img = in_type(img)
         ornt = Orientation(**init_param)
         res = ornt(img, **data_param)
         if not isinstance(res, tuple):
-            np.testing.assert_allclose(res, expected_data)
+            assert_allclose(res, expected_data, type_test=False)
             return
-        np.testing.assert_allclose(res[0], expected_data)
+        assert_allclose(res[0], expected_data, type_test=False)
```

Comment:
if the tests are created such with `p(expected_data)`, then we can remove the `type_test`, right?